arm-monitor/2017-04-01/swagger/actionGroups_API.json

```diff
@@ -0,0 +1,506 @@
+                    },
+                    "description": "The list of webhook receivers that are part of this action group."
+                }
+            },
+            "required": [
+                "groupShortName",
+                "enabled"
+            ]
+        },
+        "EmailReceiver": {
+            "description": "An email receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the email receiver. Names must be unique across all receivers within an action group."
+                },
+                "emailAddress": {
+                    "type": "string",
+                    "description": "The email address of this receiver."
+                },
+                "status": {
+                    "readOnly": true,
+                    "$ref": "#/definitions/ReceiverStatus"
+                }
+            },
+            "required": [
+                "name",
+                "emailAddress"
+            ]
+        },
+        "SmsReceiver": {
+            "description": "An SMS receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the SMS receiver. Names must be unique across all receivers within an action group."
+                },
+                "countryCode": {
+                    "type": "string",
+                    "description": "The country code of the SMS receiver."
+                },
+                "phoneNumber": {
+                    "type": "string",
+                    "description": "The phone number of the SMS receiver."
+                },
+                "status": {
+                    "readOnly": true,
+                    "$ref": "#/definitions/ReceiverStatus"
+                }
+            },
+            "required": [
+                "name",
+                "countryCode",
+                "phoneNumber"
+            ]
+        },
+        "WebhookReceiver": {
+            "description": "A webhook receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the webhook receiver. Names must be unique across all receivers within an action group."
+                },
+                "serviceUri": {
+                    "type": "string",
+                    "description": "The URI where webhooks should be sent."
+                }
+            },
+            "required": [
+                "name",
+                "serviceUri"
+            ]
+        },
+        "ReceiverStatus": {
+            "type": "string",
+            "enum": [
+                "NotSpecified",
+                "Enabled",
+                "Disabled"
+            ],
+            "x-ms-enum": {
+                "name": "ReceiverStatus",
+                "modelAsString": false
```

Comment:
That's the value (i.e. false) we use in all the other enum declarations. Maybe because we rarely change these lists.
I am not sure if @wilyle can foresee changes in this particular list.